tests/integration/components/trigger-test.js

```diff
@@ -127,10 +125,10 @@ module('Integration | Component | basic-dropdown-trigger', function (hooks) {
     assert
       .dom('.ember-basic-dropdown-trigger')
       .hasAttribute('aria-disabled', 'true', 'It is marked as disabled');
-    run(() => this.set('dropdown.disabled', false));
+    this.set('dropdown', { ...this.dropdown, disabled: false });
     assert
       .dom('.ember-basic-dropdown-trigger')
-      .doesNotHaveAttribute('aria-disabled', 'It is NOT marked as disabled');
+      .hasAttribute('aria-disabled', 'false', 'It is NOT marked as disabled');
```

Comment:
Opted to explicitly set `aria-disabled` to "true" or "false", rather than removing the attribute for false. `aria-disabled` works differently to the standard `disabled` attribute which only cares about its presence (it will still disable the element when "false", if that makes sense)